keyboards/kodeboard/keymaps/default/keymap.c

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+ /* This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#include QMK_KEYBOARD_H
+
+/* Layer Setup +for kodeboard<bukil@2024>*/
+#define _DL 0
+#define _FL 1
+#define _CL 2
+#define _ML 3
+
+/*ENCODER_DEFINE AND OP*/
+
+#define ENCODERS_PAD_A { GP3}
+#define ENCODERS_PAD_B { GP4}
+
+/*ENCODER FUNCTION FOR LYR*/
+
+
```

Comment:
```suggestion
```

This doesn't actually do anything here.  Also, this should be in the info.json/keyboard.json, not at the keymap level.